scripts/deploy.sh

```diff
@@ -28,13 +28,11 @@ npx @adobe/reactor-uploader@5.0.4 package-adobe-alloy-${VERSION}.zip \
   --org-id=97D1F3F459CE0AD80A495CBE@AdobeOrg \
   --tech-account-id=CC7A4BD95E695DBA0A495EB7@techacct.adobe.com \
   --api-key=f401a5fe22184c91a85fd441a8aa2976 \
-  --private-key="./private.key" || sleep 300
+  --private-key="./private.key" || sleep 300 \
+  --upload-timeout=300
```

Comment:
Nice! Should we remove the `sleep 300` now?

Reply:
Yes. I forgot to remove it. It is removed now.